### DISCUSSION TITLE
Request to add NASA MODIS Flood products to Data Catalog

### DISCUSSION THREAD

**Discussion by estacy** [microsoft/PlanetaryComputer#130] [Data request]
The [old NASA MODIS Flood Mapping Product](https://floodmap.modaps.eosdis.nasa.gov/index.php) spans March 2012-April 2022. This data source provides global surface water tiles on a daily, 2-day, 3-day, and 14-day basis with indicators for flooding. The product was updated in April 2022 to a new product, the [MCDWD_L3_NRT](https://www.earthdata.nasa.gov/learn/find-data/near-real-time/modis-nrt-global-flood-product). 

Both products would be useful additions to the data catalog. I am currently working with the old product and uploading my own files as needed for my analysis.

**Comment by TomAugspurger** [comment]
Thanks for opening this. We can add it to the backlog, but to set expectations it'll be some time before we could get to it.

The workaround for now is to store it yourself, as you're doing in https://github.com/microsoft/PlanetaryComputer/discussions/129. I'll a look more closely at that issue in a bit.
